src/comps/fs/dir.js

```diff
@@ -33,6 +34,8 @@ function readDataDir(dataDir) {
             }
         } else if (file.includes("rewardsinfo")) {
```

Comment:
Should we change the table name to `active_rewards_info`?

Reply:
not sure on what other changes that would require